lib/Frontend/CompilerInvocation.cpp

```diff
@@ -392,6 +392,19 @@ static void ParseModuleInterfaceArgs(ModuleInterfaceOptions &Opts,
   }
 }
 
+/// Checks if an arg is generally allowed to be included
+/// in a module interface
+static bool ShouldIncludeModuleInterfaceArg(const Arg *A) {
```

Comment:
nitpick: a more descriptive name for this function can be something like `ShouldIncludeExperimentalFeatureFlagInModuleInterface`.